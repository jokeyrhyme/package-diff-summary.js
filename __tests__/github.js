/* @flow */
'use strict'

const github = require('../lib/github.js')

test('projectUrlFor("does-not-exist")', () => {
  const name = 'does-not-exist-' + (Math.random() * 1e6)
  return github.projectUrlFor(name)
    .then((result) => {
      expect(result).not.toBeDefined()
    })
})

test('projectUrlFor("execa")', () => {
  return github.projectUrlFor('execa')
    .then((result) => {
      expect(result).toMatchSnapshot()
    })
})
