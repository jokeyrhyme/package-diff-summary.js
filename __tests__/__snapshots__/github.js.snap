exports[`test projectUrlFor("execa") 1`] = `"https://github.com/sindresorhus/execa"`;
